src/manifests/test_manifest.py

```diff
@@ -117,12 +130,30 @@ def __create__(self, data: dict) -> 'TestComponent':
         return TestComponent(data)
 
 
+class ClusterConfig:
+    def __init__(self, cluster_config_data: dict):
+        self.cluster_name = cluster_config_data['cluster_name']
+        self.data_nodes = cluster_config_data['data_nodes']
+        self.cluster_manager_nodes = cluster_config_data['cluster_manager_nodes']
+
+
+class TestComponentTopology:
+    def __init__(self, data: dict):
+        if data is not None:
+            self.cluster_configs = []
+            for cluster_config_data in data:
+                self.cluster_configs.append(ClusterConfig(cluster_config_data))
+        else:
+            self.cluster_configs = [ClusterConfig({'cluster_name': 'cluster1', 'data_nodes': 1, 'cluster_manager_nodes': 0})]  # type: ignore[assignment]
```

Comment:
This is the wrong place to provide a default, it should be in `ClusterConfig#__init__` (see above).